tasks/installer-setup.yml

```diff
@@ -0,0 +1,158 @@
+---
+- name: Generate installer trace ID
+  set_fact:
+    datadog_installer_trace_id: "{{ 9999999999999999999 | random }}"
+  when: datadog_installer_enabled
+
+- name: Start duration measurements
+  command: "date +%s%N"
+  register: datadog_installer_start_time
+  changed_when: true
+
+- name: Install datadog-installer package (dnf)
+  dnf:
+    name: "datadog-installer"
+    update_cache: true
+    state: latest # noqa package-latest
+  register: datadog_installer_install_result
+  # Since we want to send telemetry including when the installation failed,
+  # we need to explicitely ignore failures and skip dependent task when one
+  # of its depedency failed
+  # By default ansible will stop on the first error
+  ignore_errors: true
+  when: ansible_facts.os_family in ["RedHat", "Rocky", "AlmaLinux"] and not ansible_check_mode and ansible_pkg_mgr == "dnf"
+
+- name: Install latest datadog-agent package (yum)
+  yum:
+    name: "datadog-installer"
+    update_cache: true
+    state: latest # noqa package-latest
+  register: datadog_installer_install_result
+  ignore_errors: true
+  when: ansible_facts.os_family in ["RedHat", "Rocky", "AlmaLinux"] and not ansible_check_mode and ansible_pkg_mgr == "yum"
+
+- name: Install Datadog installer (apt)
+  apt:
+    name: "datadog-installer"
+    state: latest # noqa package-latest
+    update_cache: true
+    cache_valid_time: "{{ datadog_apt_cache_valid_time }}"
+  register: datadog_installer_install_result
+  ignore_errors: true
+  when: ansible_facts.os_family == "Debian" and not ansible_check_mode
+
+- name: Bootstrap the installer
+  command: /usr/bin/datadog-bootstrap bootstrap
+  register: datadog_installer_bootstrap_result
+  environment:
+    DATADOG_TRACE_ID: "{{ datadog_installer_trace_id }}"
+    DATADOG_PARENT_ID: "{{ datadog_installer_trace_id }}"
+    DD_SITE: "{{ datadog_site | default('datadoghq.com') }}"
+    DD_API_KEY: "{{ datadog_api_key }}"
+    DD_REMOTE_UPDATES: "{{ 'true' if datadog_remote_updates is defined and datadog_remote_updates else '' }}"
+    DD_APM_INSTRUMENTATION_ENABLED: "{{ datadog_apm_instrumentation_enabled }}"
+    DD_APM_INSTRUMENTATION_LANGUAGES: "{{ datadog_apm_instrumentation_libraries | join(',') }}"
```

Comment:
We have `DD_APM_INSTRUMENTATION_LIBRARIES` in the installer, I'm not sure but shouldn't we align? cc @paullegranddc

Reply:
The install-script reads both for backward compatibility purpose (actually we should too maybe?) but `DD_APM_INSTRUMENTATION_LANGUAGES` is legacy, so this should be `DD_APM_INSTRUMENTATION_LIBRARIES`